out/artifacts/web_practice_war_exploded/WEB-INF/views/driver/display_all.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: tolya
+  Date: 08/11/2022
+  Time: 11:28
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
You should [Remove autogenerated comments from JSPs](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice)

Reply:
agree